weblate/addons/xliff.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright © 2022–2023 Loïc LEUILLIOT <loic.leuilliot@gmail.com>
+#
+# This file is part of Weblate <https://weblate.org/>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+from django.utils.translation import gettext_lazy as _
+
+from weblate.addons.base import StoreBaseAddon
+from weblate.addons.forms import XLIFFCustomizeForm
+
+
+class XLIFFCustomizeAddon(StoreBaseAddon):
+    name = "weblate.xliff.customize"
+    verbose = _("Customize XLIFF output")
```

Comment:
Maybe make it XML generic? With https://github.com/translate/translate/pull/4751, the feature is added to all Lisa subclasses.

Reply:
Make sense 👍